packages/react-next/src/components/Coachmark/Coachmark.types.ts

```diff
@@ -229,21 +233,11 @@ export interface ICoachmarkStyleProps {
    */
   isCollapsed: boolean;
 
-  /**
```

Comment:
Why were these props removed? Even in a major release, we typically try not to remove props that weren't previously marked as deprecated. Though if there's a strong argument for removal you could go ahead with it and also mark the props as deprecated in OUFR to provide at least a bit of warning.

Also, if you go ahead with removing these, you should add a note about that (and any other breaking changes) to react-next's RELEASE_NOTES.md. Same for other PRs. (We just decided on this procedure yesterday, which is why you haven't heard about it before.)

Reply:
They were removed as either unused or entirely redundant. `isBeaconAnimating` was always `true`, and `isMeasured` was always the inverse of `isMeasuring`.